src/domain/orders/details/index.tsx

```diff
@@ -0,0 +1,972 @@
+                        type: "billing",
+                      })
+                    }
+                  },
+                },
+                {
+                  label: "Go to Customer",
+                  icon: <DetailsIcon size={"20"} />, // TODO: Change to Contact icon
+                  onClick: () => navigate(`/a/customers/${order.customer.id}`),
+                },
+              ]}
+            >
+              <div className="mt-6">
+                <div className="flex w-full space-x-4 items-center">
+                  <div className="flex w-[40px] h-[40px] ">
+                    <Avatar
+                      user={order?.customer}
+                      font="inter-large-semibold"
+                      color="bg-fuschia-40"
+                    />
+                  </div>
+                  <div>
+                    <h1 className="inter-large-semibold text-grey-90">
+                      {`${order?.shipping_address.first_name} ${order?.shipping_address.last_name}`}
+                    </h1>
+                    <span className="inter-small-regular text-grey-50">
+                      {order?.shipping_address.city},{" "}
+                      {order?.shipping_address.country_code}
+                    </span>
+                  </div>
+                </div>
+                <div className="flex mt-6 space-x-6 divide-x">
+                  <div className="flex flex-col">
+                    <div className="inter-small-regular text-grey-50 mb-1">
+                      Contact
+                    </div>
+                    <div className="flex flex-col inter-small-regular">
+                      <span>{order?.email}</span>
+                      <span>{order?.shipping_address?.phone || ""}</span>
+                    </div>
+                  </div>
+                  <Address title={"Shipping"} addr={order?.shipping_address} />
+                  <Address title={"Billing"} addr={order?.billing_address} />
+                </div>
+              </div>
+            </BodyCard>
+            <BodyCard
+              className={"w-full mb-4 min-h-0 h-auto"}
+              title="Raw Order"
+            >
+              <ReactJson
+                style={{ marginTop: "15px" }}
+                name={false}
+                collapsed={true}
+                src={order!}
+              />
+            </BodyCard>
+          </div>
+          <BodyCard title="Timeline" className="w-1/3">
+            <div></div>
+          </BodyCard>
+        </div>
+      )}
+      {addressModal && (
+        <AddressModal
+          handleClose={() => setAddressModal(null)}
+          handleSave={(obj) => handleUpdateAddress(obj)}
+          address={addressModal.address}
+          type={addressModal.type}
+          email={order?.email}
+        />
+      )}
+      {/* An attempt to make a reusable delete prompt, so we don't have to hold +10
+      state variables for showing different prompts */}
+      {deletePromptData.show && (
```

Comment:
This seems ok for now, eventually we could use the `use-imperative-dialog` from Zak's pr regarding product overview